scripts/run_simulations.py

```diff
@@ -56,8 +80,19 @@ def main(args):
 
         print(f"Run {simulator_path} {filepath} {metrics_dir}")
 
+        simulator_args = [simulator_path, "--config", filepath]
```

Comment:
Lets add metrics folder here
```suggestion
        simulator_args = [simulator_path, "--config", filepath, "--output-dir", metrics_dir]
```